### ISSUE TITLE
Fix S1104: Rule should not raise issue with BindableProperty

### ISSUE THREAD

**Issue by Evangelink** [SonarSource/sonar-csharp#1317]
### Description

From [SonarQube Google Group](https://groups.google.com/d/msgid/sonarqube/51281268-cba0-43dc-9467-c2313854d818%40googlegroups.com?utm_medium=email&utm_source=footer)

**Comment by omega1970** [comment]
S1104: Rule should not raise issue with public EventHandler in C#.

See guideline MS: https://msdn.microsoft.com/en-us/library/5z57dxz2(v=vs.85).aspx

**Comment by valhristov** [comment]
I tested with a very simple class with a public event and I could not see the analyzer raising an issue:
```
public class Events
{
    public event EventHandler E; // no issue here
}
```
@omega1970 Could you please share a simple code snippet that could be used to reproduce the problem?

Regarding the original issue - in general the BindableProperty (and DependencyProperty) fields are recommended to be `readonly` and this rule will not raise when the field is `readonly`. Perhaps the message should say that you need to either make the field private or readonly.
